inspirehep/utils/arxiv.py

```diff
@@ -59,8 +59,10 @@ def get_arxiv_id_from_record(record):
             if element.get("value", ""):
                 arxiv_id = element.get("value", "")
 
+    # XXX(jacquerie): should probably be removed.
```

Comment:
See discussion on gitter where we propose to use idutils and some additional checking.

```
In [6]: from idutils import normalize_arxiv

In [7]: normalize_arxiv('oai:arXiv.org:0201.21212')
Out[7]: u'arXiv:oai:arXiv.org:0201.21212'

In [8]: normalize_arxiv('arXiv:0201.21212')
Out[8]: 'arXiv:0201.21212'

In [9]: normalize_arxiv('hep/02011212')
Out[9]: u'arXiv:hep/02011212'

In [10]: normalize_arxiv('0201.21212')
Out[10]: u'arXiv:0201.21212'

In [11]: normalize_arxiv('0201.2121')
Out[11]: u'arXiv:0201.2121'
```
Note missing unicode on one and the **wrong** first case